planner/core/optimizer.go

```diff
@@ -32,23 +32,25 @@ const (
 	flagPrunColumns uint64 = 1 << iota
 	flagEliminateProjection
 	flagBuildKeyInfo
+	flagEliminateAgg
```

Comment:
how about moving it after `flagDecorrelate`?

Reply:
Seems reasonable.